### PR TITLE
CRUD for Problems

### LINKED ISSUE #1: Create an end point for viewing the problems.
As a user I want to see the question.

### PR DESCRIPTION
**Pull request by harlsh**
Closes #1 

- Added backend to the project
- Added docker, docker-compose to the project
- Backend connects to Postgres
- CRUD for "Problem" type

image

### CODE REVIEW

**Comment by harlsh** [comment]
Various end points for problems
```
GET /api/v1/problems
GET /api/v1/problems/<title-slug>
POST /api/v1/problems
```

**Comment by harlsh** [comment]
Response for
GET /api/v1/problems
```
{
      "title": "Median of Two Sorted Arrays",
      "titleSlug": "median-of-two-sorted-arrays",
      "content": "\u003cp\u003eGiven two sorted arrays \u003ccode\u003enums1\u003c/code\u003e and \u003ccode\u003enums2\u003c/code\u003e of size \u003ccode\u003em\u003c/code\u003e and \u003ccode\u003en\u003c/code\u003e respectively, return \u003cstrong\u003ethe median\u003c/strong\u003e of the two sorted arrays.\u003c/p\u003e\n\n\u003cp\u003eThe overall run time complexity should be \u003ccode\u003eO(log (m+n))\u003c/code\u003e.\u003c/p\u003e\n\n\u003cp\u003e\u0026nbsp;\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eExample 1:\u003c/strong\u003e\u003c/p\u003e\n\n\u003cpre\u003e\n\u003cstrong\u003eInput:\u003c/strong\u003e nums1 = [1,3], nums2 = [2]\n\u003cstrong\u003eOutput:\u003c/strong\u003e 2.00000\n\u003cstrong\u003eExplanation:\u003c/strong\u003e merged array = [1,2,3] and median is 2.\n\u003c/pre\u003e\n\n\u003cp\u003e\u003cstrong\u003eExample 2:\u003c/strong\u003e\u003c/p\u003e\n\n\u003cpre\u003e\n\u003cstrong\u003eInput:\u003c/strong\u003e nums1 = [1,2], nums2 = [3,4]\n\u003cstrong\u003eOutput:\u003c/strong\u003e 2.50000\n\u003cstrong\u003eExplanation:\u003c/strong\u003e merged array = [1,2,3,4] and median is (2 + 3) / 2 = 2.5.\n\u003c/pre\u003e\n\n\u003cp\u003e\u0026nbsp;\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eConstraints:\u003c/strong\u003e\u003c/p\u003e\n\n\u003cul\u003e\n\t\u003cli\u003e\u003ccode\u003enums1.length == m\u003c/code\u003e\u003c/li\u003e\n\t\u003cli\u003e\u003ccode\u003enums2.length == n\u003c/code\u003e\u003c/li\u003e\n\t\u003cli\u003e\u003ccode\u003e0 \u0026lt;= m \u0026lt;= 1000\u003c/code\u003e\u003c/li\u003e\n\t\u003cli\u003e\u003ccode\u003e0 \u0026lt;= n \u0026lt;= 1000\u003c/code\u003e\u003c/li\u003e\n\t\u003cli\u003e\u003ccode\u003e1 \u0026lt;= m + n \u0026lt;= 2000\u003c/code\u003e\u003c/li\u003e\n\t\u003cli\u003e\u003ccode\u003e-10\u003csup\u003e6\u003c/sup\u003e \u0026lt;= nums1[i], nums2[i] \u0026lt;= 10\u003csup\u003e6\u003c/sup\u003e\u003c/code\u003e\u003c/li\u003e\n\u003c/ul\u003e\n",
      "difficulty": "Hard",
      "likes": 14578,
      "dislikes": 1837,
      "exampleTestCases": "[1,3]\n[2]\n[1,2]\n[3,4]",
      "hints": ""
    }
```

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+POSTGRES_USER=leechcoder
```
Comment (yashtibrewal) [inline .env:1]: instead of pushing the .env, could you push a .env.sample
for server we can keep the naming .env

**Review by yashtibrewal — COMMENTED** [review]
Great work sir.

**Pull request merged**